squarelet/users/viewsets.py

```diff
@@ -112,6 +113,19 @@ def get_object(self):
             return super().get_object()
 
 
+class PressPassUserInvitationViewSet(
+    mixins.ListModelMixin, viewsets.GenericViewSet,
+):
+    queryset = Invitation.objects.none()
+    serializer_class = PressPassUserInvitationsSerializer
+    permission_classes = (DjangoObjectPermissions,)
+    lookup_field = "user__uuid"
+
+    def get_queryset(self):
+        user = get_object_or_404(User, uuid=self.kwargs["user_uuid"],)
```

Comment:
I think we should limit this to only allowing you to view your own invites - so return an empty query set unless the passed in uuid equals your own uuid.  Would also be useful to allow `me` for the uuid like in the base user view, for both this and user memberships.  That should make it safe to show all the invitations on the next line.

Also, reviewing this got me looking at the `PressPassUserMembershipViewSet` which shows all memberships for any user - it should be restricting those to memberships viewable for the current user using `user.memberships.get_viewable(self.request.user)` - you can throw that into this PR or ask @TylerFisher to make the change ( `git blame` tells me it is his code)